LagCompensation/scripting/LagCompensation.sp

```diff
@@ -156,12 +157,21 @@ int g_aBlockTriggerMoved[MAX_EDICTS / 32];
 int g_aBlacklisted[MAX_EDICTS / 32];
 
 Handle g_hCookie_DisableLagComp;
+Handle g_hCookie_LagCompMessages;
 bool g_bDisableLagComp[MAXPLAYERS + 1];
+bool g_bLagCompMessages[MAXPLAYERS + 1];
 int g_iDisableLagComp[MAXPLAYERS + 1];
+ConVar g_cvCompenseOnlyBosses;
+ConVar g_cvCheckUserPing;
+ConVar g_cvMinimumPing;
 
 public void OnPluginStart()
 {
-	CreateConVar("sm_lagcomp_version", PLUGIN_VERSION, "LagCompensation Version", FCVAR_SPONLY|FCVAR_NOTIFY|FCVAR_DONTRECORD).SetString(PLUGIN_VERSION);
+	g_cvCompenseOnlyBosses = CreateConVar("sm_lagcomp_onlybosses", "0", "Only lagcompensate bosses [0 = Bosses and triggers | 1 = Bosses only]", _, true, 0.0);
```

Comment:
i think this should be called  `sm_lagcomp_entityclass` with values being 0 = all, 1 = func_physbox only